codegen/model/values/helm_chart_values.go

```diff
@@ -64,8 +64,8 @@ type UserValues struct {
 	PodSecurityContext *v1.PodSecurityContext         `json:"podSecurityContext,omitempty" desc:"Pod-level security context. For more info, see the [Kubernetes documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#podsecuritycontext-v1-core)." omitChildren:"true"`
 
 	// Overrides which can be set by the user
-	DeploymentOverrides *appsv1.Deployment `json:"deploymentOverrides" desc:"Arbitrary overrides for the component's [deployment template](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/)." omitChildren:"true"`
-	ServiceOverrides    *v1.Service        `json:"serviceOverrides" desc:"Arbitrary overrides for the component's [service template](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/)." omitChildren:"true"`
+	DeploymentOverrides *appsv1.Deployment `json:"deploymentOverrides" desc:"Arbitrary overrides for the component's [deployment spec template](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/), such as to set resource limits and requests. Format the overrides as YAML or JSON." omitChildren:"true"`
+	ServiceOverrides    *v1.Service        `json:"serviceOverrides" desc:"Arbitrary overrides for the component's [service spec template](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/)." omitChildren:"true"`
```

Comment:
Imo we should remove "template" because otherwise it seems like we're talking about the `spec.template` field